packages/engine-server/src/enginev2.ts

```diff
@@ -409,11 +408,13 @@ export class DendronEngineV2 implements DEngine {
     }
   }
 
+  /**
+   * @deprecated - See {@link DendronEngineV2.findNotes}
```

Comment:
It looks like there are very few, if any, usages of this. Could we drop this entirely if it's not too hard?

Reply:
+1 on this. it should only be used by goto note and the cli. are there any other usages @tma66 ?

Reply:
I can remove it. I wasn't sure if we wanted to change the engine api as part of this change but I can go ahead and do that now